src/hotspot/share/oops/instanceRefKlass.hpp

```diff
@@ -58,6 +58,8 @@ class InstanceRefKlass: public InstanceKlass {
  public:
   InstanceRefKlass() { assert(DumpSharedSpaces || UseSharedSpaces, "only for CDS"); }
 
+  virtual bool is_instance_ref_klass()        const { return true; }
```

Comment:
Don't make this change.  There already exists an idiom for testing for reference klasses.

k->is_instance_klass() && (InstanceKlass::cast(k)->reference_type() != REF_NONE)

Reply:
Thanks, Kim! I didn't actually intend to open this up for review, and have reverted it back to 'draft' status. Thank you anyway for the suggestion: it is one of the things that I wanted to ask you guys about. I will make that change, thank you!